subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -44,7 +44,7 @@ LOG_MODULE_REGISTER(hawkbit, CONFIG_HAWKBIT_LOG_LEVEL);
 #define RECV_BUFFER_SIZE 640
 #define URL_BUFFER_SIZE 300
 #define SHA256_HASH_SIZE 32
-#define STATUS_BUFFER_SIZE 200
+#define STATUS_BUFFER_SIZE (100 + CONFIG_HAWKBIT_ADDITIONAL_STATUS_BUFFER_SIZE)
```

Comment:
I feel like it would be better to move the whole STATUS_BUFFER_SIZE to Kconfig. Otherwise, there is this weird semi-magic number.
Maybe add a recommendation to the help text so users know that they shouldn't set that value below 100

Reply:
the 100 was just the half of the default value, it can also be set below 100